platform/windows/detect.py

```diff
@@ -169,7 +169,7 @@ def get_flags():
 
 	return [
 		('freetype','builtin'), #use builtin freetype
-		('openssl','builtin'), #use builtin openssl
+		#('openssl','builtin'), #use builtin openssl
```

Comment:
```
Nick@Nick-PC MINGW64 ~/Desktop/github/godot (master)
$ git rebase -i HEAD~2
unix2dos: converting file C:/Users/Nick/Desktop/github/godot/.git/rebase-merge/git-rebase-todo to DOS format...
dos2unix: converting file C:/Users/Nick/Desktop/github/godot/.git/rebase-merge/git-rebase-todo to Unix format...
Successfully rebased and updated refs/heads/master.

Nick@Nick-PC MINGW64 ~/Desktop/github/godot (master)
$ git commit --amend
unix2dos: converting file C:/Users/Nick/Desktop/github/godot/.git/COMMIT_EDITMSG to DOS format...
dos2unix: converting file C:/Users/Nick/Desktop/github/godot/.git/COMMIT_EDITMSG to Unix format...
[master 76721d8] RichTextLabel: Add meta interactive feature
 Author: Paulb23 <p_batty@hotmail.co.uk>
 Date: Wed May 25 16:57:33 2016 +0100
 7 files changed, 41 insertions(+), 7 deletions(-)

Nick@Nick-PC MINGW64 ~/Desktop/github/godot (master)
$ git push upstream master
Username for 'https://github.com': dillybob92
To https://github.com/Dillybob92/godot.git
 ! [rejected]        master -> master (non-fast-forward)
error: failed to push some refs to 'https://github.com/Dillybob92/godot.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.
```


I'm about done with git to be honest, I'm really about to rage